applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -318,8 +347,8 @@ static void complete_dfu_data_store(void)
 static void store_dfu_data_chunk(void)
 {
 	/* Some flash may require word alignment. */
-	BUILD_ASSERT((STORE_CHUNK_SIZE % sizeof(uint32_t)) == 0);
-	BUILD_ASSERT((sizeof(sync_buffer) % sizeof(uint32_t)) == 0);
+	BUILD_ASSERT((STORE_CHUNK_SIZE % (4 * sizeof(uint32_t))) == 0);
+	BUILD_ASSERT((sizeof(sync_buffer) % (4 * sizeof(uint32_t))) == 0);
```

Comment:
Previously there was alignment to the `sizeof(uint32_t)` for all NVM and now it has been changed to the 4 times original value.
Will it still be good for all old targets?